ci/main.cue

```diff
@@ -2,98 +2,148 @@ package main
 
 import (
 	"dagger.io/dagger"
-	"dagger.io/dagger/core"
+
 	"universe.dagger.io/bash"
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/docker"
+	"universe.dagger.io/go"
+
+	"github.com/dagger/dagger/ci/pkg/golangci"
+	"github.com/dagger/dagger/ci/pkg/shellcheck"
+	"github.com/dagger/dagger/ci/pkg/markdownlint"
 )
 
 dagger.#Plan & {
-
 	// FIXME: Ideally we would want to automatically set the platform's arch identical to the host
 	// to avoid the performance hit caused by qemu (linter goes from <3s to >3m when arch is x86)
 	// Uncomment if running locally on Mac M1 to bypass qemu
 	// platform: "linux/aarch64"
 	platform: "linux/amd64"
 
-	client: filesystem: "./build": write: contents: actions.build.export.directories["/build"]
+	client: filesystem: "../": read: exclude: [
+		"ci",
+		"**/node_modules",
+		"cmd/dagger/dagger",
+		"cmd/dagger/dagger-debug",
```

Comment:
I think you could also remove *.md, docs, website etc... But that's maybe overkill, wouldn't be better to use `include` instead?

Reply:
We're linting the docs now (`actions.lint.markdown`). The plan is to replace the whole Makefile eventually